### ISSUE TITLE
Scroll doesn't work in ol.Overlay

### ISSUE THREAD

**Issue by ShahriatHossain** [openlayers/openlayers#9462]
I am using openlayers 4.6 version.
I have implemented ol.Overlay with html but scroll doesn't work. Here is my code below:

`const showInfoBox = function (e, md, host) {
       
        var rawHtml = this.ngTemplateCache.get('pointerPopupTemplate.html');
        if (!document.getElementById("infoboxContainer")) {
            $(".map__container").append("<div id='infoboxContainer' class='pointer-popup__container show'></div>");
        }
        $("#infoboxContainer").html(self.$compile(rawHtml)(host.$scope));
        var container = document.getElementById("infoboxContainer");
        this.InfoBox = new ol.Overlay({
            element: container,
            autoPan: true,
            autoPanAnimation: {
                source: null,
                duration: 250
            }
        });
        this.MapControl.addOverlay(this.InfoBox);
        
        this.InfoBox.setPosition(md.coordinate);
    };`

**Comment by tschaub** [comment]
Does the problem exist with a more recent release?

If you are able to put together an example that demonstrates the problem with a recent release or an example without additional library or application code, it would be more likely to get attention.

Starting with one of the hosted examples (e.g. https://openlayers.org/en/master/examples/overlay.html) and clicking the edit button is a good way to start.  You can paste the URL of the example that demonstrates your problem here.

**Comment by ShahriatHossain** [comment]
Basically, I have integrated my client based solution into dynamics crm's field service mobile app. Not sure it is the reason of ol overlay or the field service mobile app environment. Our solution works as offline html in field service mobile app.

**Comment by tschaub** [comment]
Here is an example of scrolling in an overlay:

![scroll](https://user-images.githubusercontent.com/41094/56762610-78ee1880-675d-11e9-9dc5-0b2595985289.gif)

In that case, the overlay element has a child with this css:
```css
#popup-content {
  max-height: 5em;
  overflow: auto;
}
```

You can see the full example working here: https://codesandbox.io/embed/q3ompw7zw6

There may indeed be an issue with scrolling and the overlay.  But I don't think the title of this ticket ("Scroll doesn't work in ol.Overlay") is accurate.  Maybe "Scrolling doesn't work in my application"?

**Comment by ShahriatHossain** [comment]
@tschaub  Thanks for your reply. We have reproduced the issue. Since we have integrated openlayers and openlayers generate some markup like below:

<div class="ol-viewport ol-touch">
<canvas class="ol-unselectable" width="1782" height="1140" style="width: 100%; height: 100%; display: block;"></canvas>
</div>

the scroll under ol overlay doesn't work because of <canvas> tag inside <div class="ol-viewport"></div> as you have seen above. So when we remove canvas from the ol-viewport div scroll works under overlay.  

We are using openlayers 4.6 version here. But we have found also that if we use openlayers current version I mean the latest one 5.x then the scroll issue gone. 

I need your suggestion now, since we are using 4.6 so using latest version may break our current code base. So what should we do now?
